pkg/agent/testdata/RawUbuntuContainerd/line62.sh

```diff
@@ -39,12 +39,13 @@ installDeps() {
         BLOBFUSE2_VERSION="2.2.0"
     fi
 
-    if [[ "${OSVERSION}" != "24.04" ]]; then
```

Comment:
There is not `templates_generated.go` changes in this PR too
Do you all know if that's expected? Is that file no longer used?

Reply:
seems expected, as discussed offline!